documentation/examples/custom-sd/adapter/adapter.go

```diff
@@ -34,6 +36,31 @@ type customSD struct {
 	Labels  map[string]string `json:"labels"`
 }
 
+//return targetsHash.Sum64() ^ labelsHash.Sum64(), nil
+func (sd *customSD) hash() (uint64, error) {
```

Comment:
I think you could leverage https://godoc.org/github.com/prometheus/common/model#LabelSet.Fingerprint which does mostly the same thing.

Reply:
Yep, for a `targetgroup` targets is `[]model.LabelSet` and labels are `model.LabelSet`

https://github.com/prometheus/prometheus/blob/master/discovery/targetgroup/targetgroup.go#L23-L33

Reply:
done